polyfills/Map/config.json

```diff
@@ -5,11 +5,14 @@
 	],
 	"browsers": {
 		"ie": "<12",
+		"safari": "<7.1",
 		"firefox": "<29",
-		"safari": "<8",
-		"ios_saf": "<7.1",
 		"chrome": "<38",
-		"firefox_mob": "<29"
+		"android": "*",
+		"firefox_mob": "<29",
+		"ie_mob": "*",
+		"opera_mob": "*",
+		"ios_saf": "<7.1"
```

Comment:
These changes come from Set/config.json. This config possibly will be working because Set/config.json is working, and already has `"android": "*"`. WeakSet's changes also come from WeakMap.